db/migrate/20170913170638_create_book_suggestion.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBookSuggestion < ActiveRecord::Migration[5.1]
+  def change
+    create_table :book_suggestions do |t|
+      t.string :editorial
```

Comment:
can we set a default empty string to the non-required fields? also null/not-null for the rest of the columns